ash/src/entry.rs

```diff
@@ -168,8 +168,10 @@ impl<L> EntryCustom<L> {
         &self.entry_fn_1_1
     }
 
-    #[deprecated = "This function is unavailable and therefore panics on Vulkan 1.0, please use `try_enumerate_instance_version` instead"]
+    #[deprecated = "This function is unavailable and therefore panics on Vulkan 1.0"]
     #[doc = "<https://www.khronos.org/registry/vulkan/specs/1.2-extensions/man/html/vkEnumerateInstanceVersion.html>"]
+    ///
+    /// Please use [`Self::try_enumerate_instance_version`] instead.
```

Comment:
Intradoc links don't work in deprecation warnings - perhaps we should leave the original function name in the deprecation message?

Reply:
I'd favor a helpful deprecation message over a link, especially since the link doesn't go very far in this case.

Reply:
Reverted the deprecation message, but left the link in docs in-place.